subscriptions/urls.py

```diff
@@ -1,9 +1,49 @@
 from django.urls import path
+from rest_framework_bulk.routes import BulkRouter
 
-from subscriptions.views import get_widgets, get_associations, set_subscriptions
+from subscriptions.views.subscription_viewset import WidgetSubscriptionViewSet
+from subscriptions.views.widget_balance import widget_balance_view
+from subscriptions.views.widget_birthday import widget_birthday_view
+from subscriptions.views.widget_library import widget_library_view
+from subscriptions.views.widget_marketplace import widget_marketplace_view
+from subscriptions.views.widget_poll import widget_poll_view
+from subscriptions.views.widget_repartition import widget_repartition_view
+from subscriptions.views.widget_timeline import widget_timeline_view
+from subscriptions.views.widget_vote import widget_vote_view
+from subscriptions.views.widgets_list import get_widgets
+
+"""
+Mandatory widgets :
```

Comment:
What do you mean by “mandatory”? Couldn't we let the user fully personalize their home page?

Reply:
I think they all have a crucial role to play in the associative live and removing one would help people disconnect with it

Reply:
OK, and where will this be enforced? In the frontend?